packages/site-content/src/eleventy.config.dev.ts

```diff
@@ -4,11 +4,12 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-const fs = require('fs');
-const pathlib = require('path');
-const eleventyNavigationPlugin = require('@11ty/eleventy-navigation');
+import fs from 'fs';
+import pathlib from 'path';
+import eleventyNavigationPlugin from '@11ty/eleventy-navigation';
+import type {EleventyConfig, EleventyConfigResult} from '@11ty/eleventy';
 
-module.exports = (eleventyConfig) => {
+module.exports = (eleventyConfig: EleventyConfig): EleventyConfigResult => {
```

Comment:
Now that these are .ts files, can this be a default export?

Reply:
If we write `export default` then the commonjs we get is:

```js
exports.default = (eleventyConfig) => ...
```

but that doesn't work with eleventy. we need:

```js
module.exports = (eleventyConfig) => ...
```